wp-content/themes/pub/wporg-learn-2024/patterns/local-navigation-bar.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Title: Local Navigation Bar
+ * Slug: wporg-learn-2024/local-navigation-bar
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:wporg/local-navigation-bar {"className":"has-display-contents","backgroundColor":"charcoal-2","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"},":hover":{"color":{"text":"var:preset|color|white"}}}}},"textColor":"white","fontSize":"small"} -->
+
+	<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"textColor":"light-grey-1","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+	<div class="wp-block-group has-light-grey-1-color has-text-color">
+		<!-- wp:site-title {"level":0,"fontSize":"small","textColor":"white"} /-->
+		<?php if ( ! ( is_page( 'learning-pathways' ) || is_archive( 'course' ) || is_archive( 'lesson' ) || is_page( 'online-workshops' ) || is_page( 'my-courses' ) ) ) : ?>
```

Comment:
If we used a more template based approach we could avoid having PHP logic here, and therefore a pattern.

Create 2 parts `local-nav.html`, `local-nav-with-title.html`, for the 2 states of this pattern.

Have 2 header parts which use them; existing `header.html` uses whatever the most common state is (looks like `local-nav-with-title.html`), and a second header part for these 4 exception templates (maybe `header-without-title.html).

A bit tricky to explain, make sense?

Reply:
yeah, this is clearer. thanks!

c344813, d33caa1